lib/trino-orc/src/test/java/io/trino/orc/TestOrcLz4.java

```diff
@@ -52,13 +48,9 @@ public void setUp()
     {
         tempFile = new TempFile();
         Random random = new Random();
-        StandardStructObjectInspector objectInspector = getStandardStructObjectInspector(ImmutableList.of("x", "y", "z"),
-                ImmutableList.of(javaLongObjectInspector, javaIntObjectInspector, javaLongObjectInspector));
-        writeOrcColumnsHive(
-                tempFile.getFile(),
-                ORC_12,
-                LZ4,
-                objectInspector,
+        writeOrcColumnsHiveFile(
+                tempFile.getFile(), ORC_12, LZ4,
```

Comment:
We either use all the parameters on one line or we use separate lines for each of the parameters.

Reply:
ok. fixed.